projects/experimental/components/button/button.template.html

```diff
@@ -9,8 +9,11 @@
     [src]="iconRight"
 ></tui-svg>
 <tui-loader
-    *ngIf="loading"
+    aria-live="polite"
+    role="status"
     class="t-loader"
     [inheritColor]="true"
+    [showLoader]="!!loading"
     [size]="loaderSize"
+    [textContent]="label"
```

Comment:
Why not `aria-*` ?

Reply:
I tried different combinations with NVDA and this works best. aria-label was spoken twice, I think.